packages/DropdownMenu/src/DropdownMenu.js

```diff
@@ -159,11 +164,12 @@ function DropdownMenu(props) {
       align={align}
       offset={popoverOffset}
       isOpen={isOpen}
+      edge={edge}
+      zIndex={zIndex}
+      {...moreProps}
```

Comment:
Moved this above `onClose` to unsure there's no way that can be overridden by the consumer.